parlai/mturk/core/mturk_utils.py

```diff
@@ -276,26 +276,34 @@ def remove_worker_qualification(worker_id, qualification_id,
 def create_hit_type(hit_title, hit_description, hit_keywords, hit_reward,
                     assignment_duration_in_seconds, is_sandbox,
                     qualifications=None,
-                    auto_approve_delay=4*7*24*3600, # default 4 weeks
+                    auto_approve_delay=4*7*24*3600,  # default 4 weeks
```

Comment:
should we consider making this shorter since people often forget to set it?

Reply:
Yeah probably, maybe I'll default 1 week instead? Will also update docs

Reply:
+1